### PR TITLE
feat(up-parachain): enable optional verbose output

### PR DESCRIPTION
**Pull request by evilrobot-01**
Adds an optional `--verbose` or `-v` argument to the up parachain command, which includes the command that was used to launch a node in the output which can be useful in advanced scenarios.

### CODE REVIEW

**Review by AlexD10S — CHANGES_REQUESTED** [review]

File: src/commands/up/mod.rs [hunk 1]
```diff
@@ -15,7 +15,7 @@ pub(crate) struct UpArgs {
 #[derive(Subcommand)]
 pub(crate) enum UpCommands {
 	#[cfg(feature = "parachain")]
-	/// Deploy a parachain to a network.
+	/// Deploy a parachain to a local network, powered by zombienet-sdk <https://github.com/paritytech/zombienet-sdk>.
```
[the remark this answers is not in the document]
Comment (Daanvdplas) [reply src/commands/up/mod.rs:18]: I agree:
Screenshot 2024-03-22 at 11 40 56

File: src/commands/up/parachain.rs [hunk 2]
```diff
@@ -21,6 +20,9 @@ pub(crate) struct ZombienetCommand {
 	/// The url of the git repository of a parachain to be used, with branch/release tag specified as #fragment (e.g. 'https://github.com/org/repository#tag'). A specific binary name can also be optionally specified via query string parameter (e.g. 'https://github.com/org/repository?binaryname#tag'), defaulting to the name of the repository when not specified.
 	#[arg(short, long)]
 	parachain: Option<Vec<String>>,
+	/// Whether the output should be verbose.
+	#[arg(short, long, action)]
```
[the remark this answers is not in the document]
Comment (Daanvdplas) [reply src/commands/up/parachain.rs:24]: Not sure, the README should explain the most important commands. There will be many commands, not sure whether we should name them all. In the docs for pop cli yes.

[the remark this answers is not in the document]
Comment (AlexD10S) [reply src/commands/up/parachain.rs:24]: Actually you are right. Resolved

File: src/commands/up/mod.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (evilrobot-01) [reply src/commands/up/mod.rs:18]: Attribution is important, this wouldn't be possible without zombienet-sdk and without it being included in the help, it makes it seem as though we built it entirely which is false.

We will be integrating a lot of tools and should give the credit to the builders.

I personally wouldn't look at the docs, expecting that once the tool is installed the help is enough to let me use it and continue to use it.

[the remark this answers is not in the document]
Comment (Daanvdplas) [reply src/commands/up/mod.rs:18]: I agree, however, the `--help` command might not be the ideal place tmo. It is intended to explain the user how to use the tool not how it is build. Credits to underlying tools are important and therefore should be explicitly mentioned in the README, comments in the source code, and perhaps even a NOTICE file.

If we are starting to use more tools then the `--help` command might get very messy and confuse users. This is the opposite of what the `--help` command is for tmo.

[the remark this answers is not in the document]
Comment (AlexD10S) [reply src/commands/up/mod.rs:18]: I agree with Daan,` --help` goal is explain how to use it.

But if we still feel is worth it to have it in the help section, let's keep it short: remove the link (just **_powered by zombienet-sdk_**) and let's add a more extensive mention it in the README.

[the remark this answers is not in the document]
Comment (evilrobot-01) [reply src/commands/up/mod.rs]: I disagree but its done.

File: README.md [hunk 3]
```diff
@@ -204,3 +205,9 @@ pop up parachain -f ./tests/zombienet.toml -p https://github.com/r0gue-io/pop-no
 
 > :information_source: Pop CLI will automatically source the necessary polkadot binaries. Currently, these will be built
 > if on a non-linux system.
+
+## Acknowledgements
+
+Pop CLI would not be possible without these awesome crates!
```
Comment (weezy20) [inline README.md:211]: nit: Feels like just `zombienet-sdk` is the just the one "crate" getting credits among many others. Suggest this be reworded to say something akin to the help: "Deployment is powered by zombienet-sdk ..."

**Review by Daanvdplas — APPROVED** [review]

**Review by AlexD10S — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (evilrobot-01) [reply README.md:211]: Done

**Review by AlexD10S — APPROVED** [review]

**Pull request merged**